packages/uswds-core/src/styles/settings/_settings-components.scss

```
@@ -123,6 +123,10 @@ $theme-in-page-nav-margin-left: 4 !default;
 $theme-in-page-nav-margin-top: 2.5 !default;
 $theme-in-page-nav-top: 4 !default;
 
+// Link
+$theme-external-link-sr-label-tab-new: "External, opens in a new tab." !default;
+$theme-external-link-sr-label-tab-same: "External, opens in current tab." !default;
```

```suggestion
$theme-external-link-sr-label-tab-same: "External." !default;
```

Since links open in the current tab by default, is it overkill to announce the default behavior?

---

Good point, added your suggestion ty!